app/Models/ServiceDatabase.php

```
@@ -149,19 +195,45 @@ public function getServiceDatabaseUrl()
         return $this->belongsTo(Service::class);
     }
 
+    public function application()
+    {
+        return $this->belongsTo(Application::class);
+    }
+
+    public function application_preview()
+    {
+        return $this->belongsTo(ApplicationPreview::class);
+    }
```

PHPMD (2.15.0)

[error] The method application_preview is not named in camelCase. (undefined)

(CamelCaseMethodName)